charts/prometheus-blackbox-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 8.9.0
+version: 8.9.1
```

As minor version of sub chart changes, this chart should change minor version too.

```suggestion
version: 8.10.0
```

---

👍